Podfile

```diff
@@ -53,10 +55,26 @@ abstract_target 'GlobalDependencies' do
         pod 'Reachability'
         pod 'SAMKeychain'
         pod 'SwiftProtobuf', '~> 1.5.0'
+        
+        target 'SessionMessagingKitTests' do
+          inherit! :complete
+          
+          pod 'Quick'
+          # FIXME: change this back to use the latest 'Nimble' once a version newer than 9.2.1 has been released
+          pod 'Nimble', :git => 'https://github.com/Quick/Nimble', :commit => 'cabe966'
+        end
       end
       
       target 'SessionUtilitiesKit' do
         pod 'SAMKeychain'
+        
+        target 'SessionUtilitiesKitTests' do
+          inherit! :complete
+          
+          pod 'Quick'
+          # FIXME: change this back to use the latest 'Nimble' once a version newer than 9.2.1 has been released
```

Comment:
Seems like 10.0.0 is out. Could you check if that release fix the issue? @mpretty-cyro

Reply:
Oh cool, I've push an update with this fixed